analyzers/its/expected/BlazorSample/BlazorSample--net7.0-S2333.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S2333",
+"message":  "'partial' is gratuitous in this context.",
```

Comment:
> UPDATE: it's because ITs run in SL context, and the other half of the class is not generated, right?

Yes, this is it, I should have added a comment about it.

Reply:
Yeah, it would be helpful. I have added it in [the PR of S6798](https://github.com/SonarSource/sonar-dotnet/issues/8275)https://github.com/SonarSource/sonar-dotnet/issues/8275, that is based on this PR.
```cs
// FN: Razor generation disabled in SL context, so in ITs too
```

I can do it in the context of https://github.com/SonarSource/sonar-dotnet/issues/8275, after this PR is merged, so that we don't create conflict issues on the rebase of 8275.

Reply:
The change was not done in https://github.com/SonarSource/sonar-dotnet/issues/8275 to prioritize delivery. This point has been added to the ["S6800 leftovers" issue](https://github.com/SonarSource/sonar-dotnet/issues/8367).